index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,700;1,300;1,400&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="style.css">
+    <title>Jubaer Maruf</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="nav">
+            <h2 class="logo">My Logo</h2>
```

Comment:
- [ ] If you notice most websites have a logo, which when clicked on, the page is redirected to the home page ☺️. Therefore, i suggest that you ensure that the logo is enclosed in `a` tag and also removed from inside the `nav` tag. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.